pkg/migrator/migrator.go

```diff
@@ -40,39 +40,39 @@ func New(client client.Client) Migrator {
 }
 func (m *migrator) Migrate() error {
 
-	cscs := &v2.CatalogSourceConfigList{}
-	err := m.client.List(context.TODO(), &client.ListOptions{}, cscs)
+	opsrcs := &v1.OperatorSourceList{}
```

Comment:
Might be worth checking if the OpSrc API exists before running this code so we don't populate 404 errors.

Reply:
This is a nit since we should only hit this error when someone overrides CVO.

Reply:
Updated the call site to not log the error if the error is `IsNotFound`.